src/openapi-generator.ts

```diff
@@ -473,10 +483,19 @@ export class OpenAPIGenerator {
       },
     };
 
-    this.pathRefs[path] = {
-      ...this.pathRefs[path],
-      ...routeDoc,
-    };
+    if (type === 'route') {
```

Comment:
Hmm personally I would separate this into two methods - `generateSingleRoute` and `generateSingleWebhook`. The reusable code can be something like `generateEndpoint` or `generatePath` :thinking:? How do you feel about that?